symfony/ux-turbo/2.19/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "conflict": {
+        "symfony/framework-bundle": "<7.2",
+        "symfony/security-csrf": "<7.2"
+    },
+    "add-lines": [
+        {
+            "file": "config/packages/framework.yaml",
+            "position": "after_target",
+            "target": "        csrf_protection:",
+            "content": "            check_header: true"
```

Comment:
Do i understand this wrong or ... will it fail for every form where Turbo is not enabled ? 

Because this is a very common scenario

Reply:
you've got it right - new projects start with turbo enable for every forms

Reply:
Ok.. bold but we'll see :) And worst case a PR is quickly done to add an option :)